AudioBatchMetadata metadata
string file_path
---
Utterance utt
